client/assets/js/main.js

```diff
@@ -1,74 +1,85 @@
-$(document).ready(init);
-
+document.addEventListener("DOMContentLoaded", init);
 
 function init(){
   handleClickEvents();
 }
 
 function handleClickEvents(){
-  $('.open').on('click', function(){ //go to stories to check, leaves landing page
-    $('#landing-container').addClass('ghostOut').fadeOut(3000);
+  const greeting = document.querySelector('.greeting');
+  const landingContainer = document.getElementById('#landing-container');
+  const open = document.querySelector('.open');
+  const signUp = document.querySelector('.signUp');
+  const logIn = document.querySelector('.logIn');
+  const formLog = document.querySelector('.formLog');
+  const formSign = document.querySelector('.formSign');
+  const instruction = document.querySelector('.instruction');
+  const sign = document.querySelector('.sign');
+
+  open.onclick = function(){ //go to stories to check, leaves landing page
+    landingContainer.classList.add('ghostOut').classList.add('fade-out-transition');
     setTimeout(function(){
-      $('#landing-container').removeClass('ghostOut')
+      landingContainer.classList.remove('ghostOut');
     },3100)
-  });
+  };
 
-  $('.experience').on('click', function(){ //go to map and camera
+  document.querySelector('.experience').onclick = function(){ //go to map and camera
     if(window.outerWidth < 750 ){
       window.open('meter-index.html','_self')
     }else{
-      $('.instruction').fadeIn();
+      instruction.classList.add('fade-in');
```

Comment:
the` .experience` class here refers to "Go to experience in the story page which is now profile.html  i'm working on. so you can delete this. So does the "`. instruction`" , I'm sorry.